### ISSUE TITLE
Cell that contains <![CDATA[xxx]]> is not interpreted

### ISSUE THREAD

**Issue by Lb-lea** [miachm/SODS#79]
Hello,
I am trying to read a file produced by Excel and the header cells contains the `<![CDATA` markup
The headers basically looks like that (albeit simplified)
```
<table:table-header-rows>
  <table:table-row table:style-name="ro1">
  <table:table-cell table:style-name="ce1" office:value-type="string">
    <text:p><![CDATA[Column 1]]></text:p>
  </table:table-cell>
  <table:table-cell table:style-name="ce1" office:value-type="string">
    <text:p><![CDATA[Column 2]]></text:p>
  </table:table-cell>
  <table:table-cell table:style-name="ce1" office:value-type="string">
    <text:p><![CDATA[Column 3]]></text:p>
  </table:table-cell>
  </table:table-row>
</table:table-header-rows>
```

However in the sods SpreadSheet created with the file, the first row only contains null column. The other rowns are fine since they don´t have the cdata tag.

I have re-saved with LibreOffice and this time the cdata markup is gone and sods reads my file just fine.

I thus conclude that the cdata is what is preventing the cell values from being loaded.
After looking it up it turns out that the cdata markup marks data that contains character that could be interpreted as xml but should not. 
Thus I think sods should be able interpret my cells as text values.

 SODS version 1.6.2 

As for providing a full file, sadly the file I have contains confidential data and  I cannot reproduce the bug myself while trying to create a clean speadsheet on Excel

**Comment by miachm** [comment]
I tried to insert a <![CDATA[Column 1]]> in a xml.

SODS returns "Column 1" which it's the expected result. Whatever your problem is, it's not related with the CData modifier.

I need a way to reproduce it... Maybe you can make a copy of the file, delete all the rows content and post it here? A minimum viable example of 1 row is good enough.

**Comment by Lb-lea** [comment]
I tried to reproduce a file. The original file is from a client so I'm not really sure how they ended up with that because if I re-save it in libreOffice so anonimise the rows the import it's fine :see_no_evil: 
So I manually edited the content.xml and re-zipped the file. Not sure if it's legit but libreOffice seems to still read my file fine so I guess so...
The first row of the SimpleOdsReader is good: `[Cell{value=Test 1, formula='null', style=, num_repeated=1}]`
But the second row is null : `[Cell{value=null, formula='null', style=, num_repeated=1}]` instead of A
[empty cell.ods](https://github.com/miachm/SODS/files/12670637/empty.cell.ods)

I hope it works (or I mean doesn´t work as well) with you. 
Thank you for your time

**Comment by miachm** [comment]
@Lb-lea should be fixed in 1.6.4
